game/scripts/vscripts/items/farming/greater_tranquil_boots.lua

```diff
@@ -61,6 +61,11 @@ function modifier_item_greater_tranquil_boots:OnCreated( event )
 
 		self:StartIntervalThink( 0.1 )
 	end
+
+	self.moveSpd = spell:GetSpecialValueFor( "bonus_movement_speed" )
```

Comment:
If you're going to change this to use self.attributes then you need to set them up inside of OnRefresh too.